nusacrowd/nusa_datasets/indotacos/indotacos.py

```diff
@@ -0,0 +1,147 @@
+
+_SOURCE_VERSION = "1.0.0"
+
+_NUSANTARA_VERSION = "1.0.0"
+
+
+class IndoTacos(datasets.GeneratorBasedBuilder):
+    """IndoNLI, a human-elicited NLI dataset for Indonesian containing ~18k sentence pairs annotated by crowd workers."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    NUSANTARA_VERSION = datasets.Version(_NUSANTARA_VERSION)
+
+    BUILDER_CONFIGS = [
+        NusantaraConfig(
+            name="indotacos_source",
+            version=SOURCE_VERSION,
+            description="indotacos source schema",
+            schema="source",
+            subset_id="indotacos",
+        ),
+        NusantaraConfig(
+            name="indotacos_nusantara_text",
+            version=NUSANTARA_VERSION,
+            description="IndoTacos Nusantara schema",
+            schema="nusantara_text",
+            subset_id="indotacos",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "indotacos_source"
+    labels = ["mengabulkan sebagian", "mengabulkan seluruhnya", "menolak", "lain-lain", "menambah pajak", "mengabulkan", "membetulkan"]
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "text": datasets.Value("string"),
+                    "nomor_putusan": datasets.Value("string"),
+                    "tahun_pajak": datasets.Value("int32"),
+                    "jenis_pajak": datasets.Value("string"),
+                    "tahun_putusan": datasets.Value("int32"),
+                    "pokok_sengketa": datasets.Value("string"),
+                    "pokok_sengketa": datasets.Value("string"),
+                }
+            )
+        elif self.config.schema == "nusantara_text":
+            features = schemas.text_features(self.labels)
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        url = _URLS["indotacos"]
+        # path = dl_manager.download_and_extract(url) + "/indonesia_tax_court_verdict.csv"
+
+        # data_files = {"train": "indonesia_tax_court_verdict.csv"}
+
+        path = dl_manager.download(url)["indotacos"]
+        data_files = {"train": path}
+        # path = data
+
+        print(path)
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": data_files["train"],
+                },
+            )
+        ]
+
+    def _generate_examples(self, filepath: Path):
+        df = pd.read_csv(filepath)
+        # print(df)
+        if self.config.schema == "source":
+            for row in df.itertuples():
+                # print(row)
+                ex = {
+                    "id": str(row.Index),
```

Comment:
Based on this comment

> ### Run result
> Check test log here: https://github.com/IndoNLP/nusa-crowd/actions/runs/3122928256

I think this got error because you didn't set `id` field on source schema config in line `80`